test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -480,8 +476,13 @@ func createPods(namespace string, cpo *createPodsOp, clientset clientset.Interfa
 // waitUntilPodsScheduledInNamespace blocks until all pods in the given
 // namespace are scheduled. Times out after 10 minutes because even at the
 // lowest observed QPS of ~10 pods/sec, a 5000-node test should complete.
-func waitUntilPodsScheduledInNamespace(podInformer coreinformers.PodInformer, name string, namespace string, wantCount int) error {
+func waitUntilPodsScheduledInNamespace(ctx context.Context, podInformer coreinformers.PodInformer, name string, namespace string, wantCount int) error {
 	return wait.PollImmediate(1*time.Second, 10*time.Minute, func() (bool, error) {
+		select {
+		case <-ctx.Done():
+			return true, nil
```

Comment:
`return true, ctx.Err()`?

Reply:
No, returning an error would cause wait.PollImmediate to try again after 1 second.

Reply:
Actually, nevermind, you're right :D I really thought returning error would poll again.

Done.